storm-server/src/main/java/org/apache/storm/scheduler/multitenant/NodePool.java

```diff
@@ -98,7 +98,7 @@ public RoundRobinSlotScheduler(TopologyDetails td, int slotsToUse,
       }
       
       _spreadToSchedule = new HashMap<>();
-      List<String> spreadComps = (List<String>)td.getConf().get(Config.TOPOLOGY_SPREAD_COMPONENTS);
+      List<String> spreadComps = (List<String>)td.getConf().get(Config.TOPOLOGY_RAS_CONSTRAINT_SPREAD_COMPONENTS);
```

Comment:
I don't know about changing TOPOLOGY_SPREAD_COMPONENTS to TOPOLOGY_RAS_CONSTRAINT_SPREAD_COMPONENTS.  It seems kind of weird to me that the multitenant scheduler would have a config that references RAS.

Reply:
good point I forgot that I moved that in config instead of adding it.